Dockerfile

```diff
@@ -381,57 +381,56 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
 # Python
 ENV PYENV_ROOT=/opt/python
 ENV PATH=$PATH:${PYENV_ROOT}/shims:${PYENV_ROOT}/bin
-COPY --from=python ${PYENV_ROOT} ${PYENV_ROOT}
+COPY --from=python --chown=$USER:$USER ${PYENV_ROOT} ${PYENV_ROOT}
 #RUN sudo chown -R ${USER}.${USER} ${PYENV_ROOT}
```

Comment:
Can you please drop this out-commented line along the way?

Reply:
Removed